mermaid/stats_internal_test.go

```diff
@@ -0,0 +1,119 @@
+package mermaid
+
+import (
+	"testing"
+
+	"github.com/magiconair/properties/assert"
+)
+
+func Test_flow_prepareBasic(t *testing.T) {
+
+	flows := []*flow{
+		{
+			from: &node{
+				name: "A",
+				description: &shape{
+					text:      "First node",
+					shapeType: "[]",
```

Comment:
just saying, but in UML diagrams it is common to start and end with circle nodes, and have square in the middle